Tests/test_utils.py

```diff
@@ -18,11 +18,13 @@
     [(0, True), (-1, True), (1, True), ("1", False), (1.5, False)],
 )
 def test_isInt(value, expected):
-    assert PyPDF2._utils.isInt(value) == expected
+    with pytest.warns(UserWarning):
```

Comment:
while you're at it, you might want to specify the `match` keyword argument of `pytest.warns` to match (regexp) the error specific error message you want to raise. This avoids accidentally capturing a different error, and it makes the test much easier for yourself to maintain in the future